docs/updates/2019-01.md

```diff
@@ -0,0 +1,62 @@
+# January 2019
+
+New year, new features, new solutions. Welcome to the first update of the year, the January 2019 update! There are a
+couple of important updates which we hope you will like.
+
+## Data model experience
+
+Our data modelling tool, catwalk, has reached a maturity level where we feel comfortable to move it from experimental
+to stable. Since it has continuously been published on [catwalk.core.qlik.com](https://catwalk.core.qlik.com)
+you will, if you visited catwalk before, already feel familiar with the look and feel. Here are some of our latest
+additions:
+
+### Tool walk-through
+
+The first time you enter the tool you will be greeted with a guide walking you through the different parts of the
+application. It will describe how you can analyse your data model, how you interpret the different colours, associations
+lines and what the numbers mean.
+
+![screenshot](../images/catwalk-walk-through.png)
+
+The guide is based on your chosen application and if you would like to pull it up again in the future you can do so by
+right click on the catwalk logo (top right in the application).
+
+### More field descriptions
+
+Another feature we added to provide a better understanding of the data model is extended field information. Previously
+you could only reach field information through hoovering over the field card to view a condensed version of the field
+data. With the new feature you can also click on the three dots in the top right corner of a field to get a more
+descriptive information card with explanation.
+
+![screenshot](../images/catwalk-field-info.png)
+
+## ABAC moving to stable with a breaking change
+
+The API support for Attribute Based Access Control (ABAC) in engine has been around for a while and are ready to be
```

Comment:
Maybe link to the ABAC documentation here for better understanding of which feature being promoted?

https://core.qlik.com/services/qix-engine/access-control/

Reply:
Also not sure if we should call an `API support`, perhaps only `The feature for...`.

Reply:
Adding link!

Reason for calling in API support for is that ABAC seem to be a generic concept and not a specific feature for us